cves/CVE-2014-1418.yml

```diff
@@ -274,7 +309,15 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    This vulnerability was one that was obscure and difficult to notice, but
+    caused a pretty big potential issue. It went unnoticed for nearly 6 years,
+    but the conditions that allowed it to be exploited were rather rare especially
+    as time went on and less and less people used the older versions of IE that
+    caused the issue. If there were unit tests written when the code was first
+    added (instead of 3 years later), that might have helped. In addition, code
+    reviews could have helped to idenify the issue, as someone else looking at
+    the code could help idenity a potential vulnerability being introduced.
```

Comment:
Do you think a developer could reasonably have realized that this vulnerability was possible just by looking at the code, without actually trying to test the feature on an older version of IE?